packages/web-components/src/stories/toast.stories.mdx

```diff
@@ -0,0 +1,205 @@
+import { extractArgTypes } from '@astrouxds/storybook-addon-docs-stencil'
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import { html, render } from 'lit-html'
+const args = extractArgTypes('rux-toast')
+args.message.defaultValue =
+    'This is a toast.'
+
+<Meta
+    title="Components/Toast Stack/Toast"
```

Comment:
can you copy all the stuff from Timeline about Beta and move it over here? the beta/styleddiv and move the story under the Beta level?